libdnf/comps/environment/query.cpp

```diff
@@ -43,10 +44,11 @@ namespace libdnf::comps {
 EnvironmentQuery::EnvironmentQuery(const BaseWeakPtr & base) : base(base) {
     libdnf::solv::Pool & pool = get_pool(base);
 
-    std::map<std::string, std::vector<Id>> available_map;
+    std::map<std::string, std::vector<std::pair<Id, std::string_view>>> available_map;
```

Comment:
It might be helpful to document what's what in the container structure, the code is a bit hard to read if one doesn't know.